modules/integration_azure-container-instance/conf/00-hearbeat.yaml

```diff
@@ -0,0 +1,11 @@
+module: Azure Container Instance
+name: heartbeat
+filtering: "filter('resource_type', 'Microsoft.ContainerInstance/containerGroups') and filter('primary_aggregation_type', 'true')"
+transformation: false
+aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

Comment:
Would aggregation by `containerName` be a good idea ? I don't really know about ACI but dimension seems available.

Reply:
I think it will not have any impact on alert body but it will add this to dimensions list in its subject.

so it seems to be a good idea but it could be undesired if `azure_resource_name == containerName`.

This is up to the author

Reply:
ACI is a container group so resources are shared between them. The heartbeat detector is about a metric from all containers of the ACI so I don't think aggregation by `containername` would be pertinent.